live-build/config/hooks/vm-artifacts/90-raw-disk-image.binary

```diff
@@ -275,8 +276,8 @@ zfs create \
 # contents. During normal boot up, we'll rely on "/etc/fstab" to handle
 # these mounts.
 #
-mkdir -p "$DIRECTORY/export/home"
-mount -t zfs "$FSNAME/ROOT/$FSNAME/home" "$DIRECTORY/export/home"
+mkdir -p "$DIRECTORY/home"
+mount -t zfs -o nodev "$FSNAME/ROOT/$FSNAME/home" "$DIRECTORY/home"
```

Comment:
There is no need to mount with `nodev` here. The important part is setting it in `/etc/fstab` so that it gets set correctly on the created engine.